packages/fxa-payments-server/Gruntfile.js

```diff
@@ -3,17 +3,25 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 module.exports = function (grunt) {
-  const srcPaths = [
-    // 'src/branding.ftl' is temporary
-    // and will be replaced with '../fxa-shared/lib/l10n/branding.ftl'
-    // in a later ticket - will require coordination with l10n to resolve
-    // conflicting IDs for identical terms.
-    'src/branding.ftl',
+  const srcPaths = ['.license.header', 'src/**/*.ftl'];
+  const testPaths = [
+    '../../libs/shared/l10n/src/lib/branding.ftl',
```

Comment:
Doesn't the copy below already take care of this path?

Reply:
TBH, I copied from `fxa-settings` and don't know the answer to this. I'm less familiar with our `l10n` setup. I can update. Please let me know!